app/views/partials/_comments.html.erb

```diff
@@ -8,6 +8,12 @@
       <div class="comment">
           <span class="comment-author"><%= comment.author.name %>:</span>
           <span><%= comment.text %></span>
+
+          <% if can? :destroy, comment %>
+              <%= link_to user_post_comment_url(author_id: comment.post.author.id, post_id: comment.post.id, id: comment.id), method: :delete, data: { confirm: "Are you sure?" } do %>
+                  <button class="short-button">Delete</button>
+              <% end %>
+          <% end %>
```

Comment:
- [ ] I noticed the delete operation for comments is not functioning as it should. Kindly work on it so a comment can be deleted as expected 👍🏽

Reply:
Fixed.